Formula/trurl.rb

```diff
@@ -0,0 +1,20 @@
+class Trurl < Formula
+  desc "Command-line tool for URL parsing and manipulation"
+  homepage "https://github.com/curl/trurl"
+  url "https://github.com/curl/urler/archive/refs/tags/trurl-0.2.tar.gz"
+  sha256 "acef7d251fc13fc97153d5d6949b50744a5bad8aa93840b0d8509cd754ed704f"
+  license "curl"
+  head "https://github.com/curl/trurl.git", branch: "master"
+
+  uses_from_macos "curl"
```

Comment:
On Big Sur:

```
trurl.c:65:16: error: use of undeclared identifier 'CURLUPART_ZONEID'
  {"zoneid",   CURLUPART_ZONEID},
               ^
trurl.c:343:18: error: use of undeclared identifier 'CURLUE_NO_ZONEID'
            case CURLUE_NO_ZONEID:
                 ^
```

The symbol `CURLUE_NO_ZONEID` was [added in 7.81.0](https://github.com/curl/curl/blob/master/docs/libcurl/symbols-in-versions#L1088). (`CURLUPART_ZONEID` was [added in 7.65.0](https://curl.se/libcurl/c/curl_url_set.html).)

```suggestion
  uses_from_macos "curl", since: :ventura # uses CURLUE_NO_ZONEID, available since curl 7.81.0
```